assets/components/tracking/connector/ConnectorForm.tsx

```diff
@@ -63,173 +63,190 @@ export function ConnectorForm({form, onCreate}: { form: FormInstance, onCreate:
             </Form.Item>
 
             {
```

Comment:
This entire form's getting really fat. You might wanna split it between commons and registrar-specific parts.